notebooks/02_WriteFinalWaterbodies.ipynb

```diff
@@ -0,0 +1,681 @@
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>POLYGON ((-1119990.000 889679.999, -1119960.00...</td>\n",
+       "      <td>44100.0000</td>\n",
+       "      <td>e9pxzb5ze</td>\n",
+       "      <td>4</td>\n",
+       "      <td>1680.0</td>\n",
+       "      <td>https://deafrica-waterbodies-dev.s3.af-south-1...</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                            geometry      area_m2        UID  \\\n",
+       "0  POLYGON ((-1087200.000 849749.999, -1087080.00...   11700.0000  e9pvzx5t2   \n",
+       "1  POLYGON ((-1087140.000 850709.999, -1087080.00...  147599.9999  e9pvzxcs1   \n",
+       "2  POLYGON ((-1085580.000 849989.999, -1085580.00...    6300.0000  e9pvzzx9v   \n",
+       "3  POLYGON ((-1119570.000 888719.999, -1119570.00...   11700.0000  e9pxxzp4y   \n",
+       "4  POLYGON ((-1119990.000 889679.999, -1119960.00...   44100.0000  e9pxzb5ze   \n",
+       "\n",
+       "   WB_ID  perim_m                                         timeseries  \n",
+       "0      0    780.0  https://deafrica-waterbodies-dev.s3.af-south-1...  \n",
+       "1      1   4260.0  https://deafrica-waterbodies-dev.s3.af-south-1...  \n",
+       "2      2    360.0  https://deafrica-waterbodies-dev.s3.af-south-1...  \n",
+       "3      3    720.0  https://deafrica-waterbodies-dev.s3.af-south-1...  \n",
+       "4      4   1680.0  https://deafrica-waterbodies-dev.s3.af-south-1...  "
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "waterbodies_gdf.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "43062616-d77d-47c6-8e2c-005d114dd6cd",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "'https://deafrica-waterbodies-dev.s3.af-south-1.amazonaws.com/0-0-1/timeseries/e9pv/e9pvzx5t2_v0.csv'"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "waterbodies_gdf.iloc[0].timeseries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "1aede9ef-3a24-4b89-bcab-f247f6eb66a2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Reproject to EPSG:4326\n",
+    "waterbodies_gdf_4326 = waterbodies_gdf.to_crs(\"EPSG:4326\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "c2363caf-d318-4dcf-a50c-9e995503ad43",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[2023-10-23 11:39:25,889] {collection.py:558} WARNING - Value 129574799.878199995 of field area_m2 of feature 38 not successfully written. Possibly due to too larger number with respect to field width\n",
```

Comment:
Has this warning been addressed? Can you please check that the area calculation is working as expected? I would have expected it to be an integer multiple of a Landsat pixel area (900m^2)

Reply:
The area calculation is working as expected. The warning is shown when writing to a shapefile, but it still writes the correct value.